cfg/cfg.go

```diff
@@ -82,7 +85,30 @@ func init() {
 	}
 	flag.StringVar(&AuthenticatedUserID, "authenticated-user-id", defaultAuthenticatedUserID, "OpenShift user's ID that should has access to API. Is used only if useBearerToken is configured")
 
-	flag.DurationVar(&IdleTimeout, "idle-timeout", -1*time.Nanosecond, "IdleTimeout is a inactivity period after which workspace should be stopped. Examples: -1, 30s, 15m, 1h")
+	defaultIdleTimeout := -1 * time.Nanosecond
+	idleTimeoutEnv := "SECONDS_OF_DW_INACTIVITY_BEFORE_IDLING"
+	idleTimeoutEnvValue, isFound := os.LookupEnv(idleTimeoutEnv)
+	if isFound && len(idleTimeoutEnvValue) > 0 {
+		if v, err := strconv.Atoi(idleTimeoutEnvValue); err == nil {
+			defaultIdleTimeout = time.Duration(v) * time.Second
```

Comment:
I think, it will be the same result `v * time.Second`

Reply:
If I do:
```
	if v, err := strconv.ParseInt(idleTimeoutEnvValue, 10, 32); err == nil {
		defaultIdleTimeout = v * time.Second
	}
```
this gives me an error: 
```
invalid operation: v * time.Second (mismatched types int64 and time.Duration)
```